src/game/action/Action_Left_Right.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "IAction.h"
+#include "EntityManager.h"
+
+
+
+/**
+ * @brief Action to cause the provided entity to jump.
```

Comment:
I need to stop copying and pasting stuff around and then forgetting to modify :)